Cargo.toml

```diff
@@ -45,7 +45,7 @@ elliptic-curve = { version = "0.13.4", default-features = false }
 eth-keystore = "0.5.0"
 fuel-abi-types = "0.2.1"
 fuel-asm = "0.31.1"
-fuel-core = { version = "0.18.2", default-features = false }
+fuel-core = { version = "0.18.2", default-features = false,  features = ["rocksdb"] }
```

Comment:
this will likely increase build times a lot, might be worth putting under a feature flag?

Reply:
ok